### DISCUSSION TITLE
"Avoid crossing perimeters" vs "Avoid Printed Parts When Traveling"

### DISCUSSION THREAD

**Discussion by ghost** [supermerill/SuperSlicer#1473] [Q&A]
I just learned today that PrusaSlicer version 2.3.0 and later have changed the way that "Avoid crossing perimeters" works and it no longer works like the Cura setting "Avoid Printed Parts When Traveling".  I had recently upgraded and noticed that the tops of parts were being damaged, and saw that that travel paths between parts are now straight lines that cross printed parts, and that this change in functionality had been requested.  So, I can no longer use PrusaSlicer.

That being said, I heard about SuperSlicer a few weeks ago from Nero 3DP, and I've been considering moving to it.  However, before I start using it, I need to know if SuperSlicer is going to switch to the same functionality that PrusaSlicer has moved to?  The ability to "Avoid Printed Parts When Traveling" is a required feature, and while I don't like Cura nearly as much, I'll be forced to go back to it if this functionality is lost.

**Comment by neophyl** [comment]
As its based on PS then I would say the functionality would match PS unless specifically modified to be different by Supermerill.  Of course you could always like actually open a feature request in issues and ask that the functionality is changed.  
Actually why didn't you open one on the PS github and ask for that if its important to you ?

**Comment by ghost** [comment]
@neophyl that is not an answer, that is solely an uninformed guess on your part.  You are also making the incorrect assumption that I have not contacted the PrusaSlicer team, which is a false assumption.  I am waiting for an official response here, or I might just create another Issue to get an answer there.

**Reply by supermerill** [reply]
neophyl is very aware of what's going on. You shouldn't dismiss his opinion.

By saying "I can no longer use PrusaSlicer." and saying nothing about a possible issue created on the prusaslicer github, it's fair to assume that you haven't.

As neophyl said, I follow the changes of prusaslicer.I try to retain some old things if I found the new one not complete, but the old "avoid perimeter" was constantly trashed on by everyone and had abysmal performance, so I didn't blink an eye when they replaced it.

If you have a feature request, you should open an issue, yes.

**Reply by ghost** [reply]
The PrusaSlicer team already said no, so when you merge in those changes, SuperSlicer will not work either.  Thankyou, but I'll go back to Cura.  If the PrusaSlicer team changes their mind in the future, and adds the functionality back as it was originally intended (and works in Cura), I might reconsider.  Best...
